content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -68,11 +83,13 @@ job-banana.yaml
 job-cherry.yaml
 ```
 
-Here, we used `sed` to replace the string `$ITEM` with the loop variable.
-You could use any type of template language (jinja2, erb) or write a program
-to generate the Job objects.
+Here, you used `sed` to replace the string `$ITEM` with the loop variable.
+You could use any type of template language (for example: Jinja2; ERB), or  you
```

Comment:
nit: Here `sed` replaces
You can use any ... or write a program

Reply:
I reworded this. Hope it's OK now.